### PR TITLE
[planet-finance] UPDATE_VOTING_STRATEGY

### PR DESCRIPTION
**Pull request by ash-greninja-poke**
Fixes # .

Changes proposed in this pull request:
- 
- 
- ADDED NEW LP PAIRS VAULTS
- ADDED AQUA LENDING

### CODE REVIEW

**Comment by ChaituVR** [comment]
Hello @ash-greninja-poke we can't update this strategy as it is used by planet-finance and can affect their voting
You can always create a new strategy if these changes are important to you

**Comment by ash-greninja-poke** [comment]
I am the developer of planet-finance @ChaituVR  and owner of project told me to update the strategy.

**Comment by planetfinance** [comment]
Hey @ChaituVR can we accept this strategy, @ash-greninja-poke is part of the project and is trying to fix this.

File: src/strategies/planet-finance/index.ts [hunk 1]
```diff
@@ -155,6 +211,25 @@ export async function strategy(
...
+    'aquaGammaTotalSupply',
+    aquaGammaLpTokenAddress,
+    'totalSupply'
+  );
+
+  erc20Multi.call('lpAquaBal', aquaAddress, 'balanceOf', [
+    aquaGammaLpTokenAddress
+  ]);
+
+  erc20Result = await erc20Multi.execute();
```
Comment (ChaituVR) [inline src/strategies/planet-finance/index.ts:225]: Hello @ash-greninja-poke These changes are adding many requests, we have a limit of 5 requests 
https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

Can you check if there is a way to get all these in a single request

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ash-greninja-poke** [comment]
ok @ChaituVR will try to do that

**Pull request closed without merging**